documentation/index.rst

```diff
@@ -8,6 +8,7 @@ Documentation for GMT is available online in different versions (if in doubt, us
 
 * `Latest stable release <https://docs.generic-mapping-tools.org/latest>`__
 * `Current development version (unstable nightly builds) <https://docs.generic-mapping-tools.org/dev/>`__
+* `GMT 6.2.0rc1 <https://docs.generic-mapping-tools.org/6.2.0rc1/>`__
```

Comment:
The dropdown at https://docs.generic-mapping-tools.org/latest/ links to https://docs.generic-mapping-tools.org/6.2rc1 which doesn't work. I fumbled around and found that it points to https://docs.generic-mapping-tools.org/6.2.0rc1/.

Reply:
It's not ideal that there's no version switch at that page. I am going to submit a small docs PR, in case a new trigger of the docs workflow will fix the problem after #5162 was merged.

Reply:
Ah ok, just noticed it's been reported at https://github.com/GenericMappingTools/gmt/issues/5161.

Reply:
I have not been able to fully fix the version switch, but since this link works it's fine to merge this first.